.gitmodules

```diff
@@ -63,7 +63,7 @@
 	url = https://github.com/zdevito/sleef
 [submodule "third_party/ideep"]
 	path = third_party/ideep
-	url = https://github.com/intel/ideep
+	url = https://github.com/bddppq/ideep
```

Comment:
This is not meant for final commit, just so I can patch ideep with https://github.com/bddppq/ideep/commit/190d1ab6eb4ed2581ba02b697f7367097e07f5ea (to add `ideep::convolution_forward::expected_src_descriptor`) and verify.